packages/dd-trace/src/sampling_rule.js

```diff
@@ -0,0 +1,130 @@
+'use strict'
```

Comment:
Just to confirm, is this feature not landing in the v4.x branch? @Qard @khanayan123

Reply:
this feature already landed on v4.x and v3.x. This PR rebased on top of master but v5.x is behind master which is why this commit is showing up